app/models/drug.rb

```diff
@@ -0,0 +1,37 @@
+class Drug < ActiveYaml::Base
+  set_root_path "config/data"
+  set_filename "drugs"
+
+  CREATED_TIME ||= File.ctime("config/data/drugs.yml").in_time_zone("UTC")
+  UPDATED_TIME ||= File.mtime("config/data/drugs.yml").in_time_zone("UTC")
+
+  CATEGORIES = {
+    hypertension_ccb: "Hypertension: CCB",
+    hypertension_arb: "Hypertension: ARB",
+    hypertension_diuretic: "Hypertension: Diuretic",
+    hypertension_ace: "Hypertension: ACE Inhibitor",
+    hypertension_other: "Hypertension: Other",
+    diabetes: "Diabetes",
+    other: "Other"
+  }.freeze
+
+  FREQUENCIES = {
+    once_a_day: "Once a day",
+    twice_a_day: "Twice a day",
+    thrice_a_day: "Thrice a day"
+  }.freeze
+
+  def updated_at
+    UPDATED_TIME
+  end
+
+  def created_at
+    CREATED_TIME
+  end
+
+  def deleted_at
+    if deleted == true
+      UPDATED_TIME
+    end
+  end
```

Comment:
These methods will allow us to look up `drug.updated_at` etc and get a useful value back.

Reply:
Do we need `.updated_at` for anything other than the process token?

Reply:
No, but it's worth adding timestamps to the data model now, so we can transition to a AR backed model seamlessly (We'd need a version bump to add in this field later).